mk/spksrc.service.installer

```diff
@@ -291,7 +291,7 @@ postinst ()
         # Create share if does not exist
         #  !"#$%&’()*+,/:;<=>?@[]nˆ`{} |
         if ! synoshare --get "${SHARE_NAME}" &> /dev/null ; then
-            synoshare --add "${SHARE_NAME}" "${SHARE_DESC}" "${SHARE_PATH}" "" "rw" "" 1 0 >> ${INST_LOG}
+            synoshare --add "${SHARE_NAME}" "${SHARE_DESC}" "${SHARE_PATH}" "" "$USER" "" 1 0 >> ${INST_LOG} 2>&1
```

Comment:
As a user "rw" does not exist, the current implementation does nothing.

The access to the created share is defined below, 
either by the user
```
   synoshare --setuser "${SHARE_NAME}" RW + "${EFF_USER}" >> ${INST_LOG} 2>&1
```

or by a group:
```
   set_syno_permissions "${SHARE_PATH}" "${GROUP}"
```